plugins/client/graphql-kotlin-client-generator/src/main/kotlin/com/expediagroup/graphql/plugin/client/generateClient.kt

```diff
@@ -42,7 +47,19 @@ fun generateClient(
         customScalarMap = customScalars,
         serializer = serializer
     )
-    val graphQLSchema = SchemaParser().parse(schema)
+    val graphQLSchema = SchemaParser().parse(getSchemaReader(schema))
     val generator = GraphQLClientGenerator(graphQLSchema, config)
     return generator.generate(queries)
 }
+
+private fun getSchemaReader(schemaPath: String): Reader =
+    if (schemaPath.startsWith("classpath://")) {
```

Comment:
`classpath://` is not a valid URL protocol so we should not code against this (yes Spring supports it but it is non-standard)